module/raid/raid.py

```diff
@@ -126,7 +129,7 @@ def pt_ocr(raid):
         return None
 
 
-class Raid(MapOperation, Combat, CampaignEvent):
+class Raid(MapOperation, GuildCombat, CampaignEvent):
```

Comment:
Did this for quickness but was wondering should make a `RaidCombat` equivalent? Or should update `Combat` with the extra clauses since it is backwards compatible but maybe the assets are too similar to others?

Reply:
Make a `RaidCombat`